### DISCUSSION TITLE
cargo audit

### DISCUSSION THREAD

**Discussion by mcandre** [community/community#14100] [Code Security] [answered]
Now that there is auditing support for Rust projects with the third party cargo-audit plugin, will dependabot make use of this to scan Rust projects?

**Comment by jhutchings1** [comment, marked as answer]
Stand by for more on this. We've already got support for Rust in the [Advisory Database](https://github.com/advisories) and Dependabot version updates. We've got one more place to add support before Rust will be supported end-to-end.
